### ISSUE TITLE
OC: Make suggested actions more accessible through TAB keys or accesskey attribute

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#3354]
## Feature Request

Today, when suggested actions pops up, the user would need to press <kbd>SHIFT</kbd> + <kbd>TAB</kbd> key to move to the (last) suggested actions and move their way up.

This is not intuitive and natural.

Tomorrow, we should make this more intuitive. But unfortunately, we cannot move the focus programmatically because it would interfere/interrupt the screen reader.

Few things we could consider

- Move suggested actions after the send box
   - This could change the UI
- Add `accesskey` attribute to the suggested actions container
   - Localization is not trivial as different browsers/platforms use different modifier keys

## Demo

![recording (40)](https://user-images.githubusercontent.com/1622400/88867917-a37e1b00-d1c3-11ea-9b6c-acaa244e86b7.gif)

### Access key modifiers

![image](https://user-images.githubusercontent.com/1622400/88872296-b4805980-d1ce-11ea-9fea-f3d634c66514.png)

> Excerpt from https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/accesskey.

### Access key inconsistencies

- Depends on browsers, the element could be focused or activated (e.g. clicked)
- Depends on browsers, multiple elements with same `accesskey` values may have the first or last element triggered
   - As a component, we might conflict with the page host. We should only capture the access key when the user put focus on our UI
- Older browsers tends to deviate on modifier keys
   - Modern browsers generally converge to <kbd>ALT</kbd> + <kbd>SHIFT</kbd> on Windows and <kbd>CTRL</kbd> + <kbd>ALT</kbd> on Mac

> More read at https://webaim.org/techniques/keyboard/accesskey.

### `code` vs. `key` vs. `keyCode`

Naturally, `key` should be the better choice because:

1. It align with `accesskey` attribute
1. It is easy to understand what to fill in (probably `"a"` or `"A"`)

On Mac, when pressing <kbd>ALT</kbd> + <kbd>OPTION</kbd> + <kbd>A</kbd>, the `event.key` will return `"Å"` and `"å"` on `key` property. This is because, on Mac, <kbd>ALT</kbd> + <kbd>A</kbd> will be typed out as `"Å"` and `"å"`.

`event.code` do not take keyboard layout into account. For example, on a QWERTZ keyboard, press <kbd>Z</kbd> key will yield `KeyY` instead of `KeyZ`.

For `event.keyCode`, we cannot easily turn it into a localized string. And it is also deprecated by `event.code`.

So we end up need to use `key` and support multiple access keys, namely `"A a Å å"`.

[Enhancement]
